clients/admin-ui/src/theme/index.ts

```diff
@@ -8,6 +8,13 @@ const theme = extendTheme({
       body: {
         bg: "white",
         minWidth: "container.lg",
+        height: "100%",
+      },
+      html: {
+        height: "100%",
+      },
+      "#__next": {
```

Comment:
clever! This div has caused issues for me in the past.

Reply:
ah yes, I spent much time chasing down all the heights that needed to be 100%...